src/commands/gate/evaluate.ts

```diff
@@ -58,19 +64,22 @@ export class GateEvaluateCommand extends Command {
   }
 
   private async evaluateRules(api: APIHelper, evaluateRequest: Payload) {
+    this.context.stderr.write(renderGateEvaluation(evaluateRequest.spanTags))
     if (this.dryRun) {
-      // TODO All writes will be moved to renderer in next iterations
-      this.context.stderr.write('Dry run mode is enabled. Not evaluating the rules.')
+      this.context.stderr.write(renderDryRunEvaluation())
 
       return
     }
 
-    // To be extended
-    try {
-      await api.evaluateGateRules(evaluateRequest, this.context.stdout.write.bind(this.context.stdout))
-    } catch (error) {
-      this.context.stderr.write(error)
-    }
+    // To be extended with retries, error handling, etc.
+    await api
+      .evaluateGateRules(evaluateRequest, this.context.stdout.write.bind(this.context.stdout))
```

Comment:
this will eventually change the exit code based on the response, right? But that's a follow up PR I guess

Reply:
Yup, that's the case. I will change this behavior in the next PRs